### ISSUE TITLE
Tuple is not implemented

### ISSUE THREAD

**Issue by mtshiba** [erg-lang/erg#34]
Reproducible code:

```console
> erg examples/tuple.er
```

Result:

```
Error[#0000]: File examples/tuple.er, line 1
1│ p = (1, 2)
         ^
SyntaxError: invalid syntax
Error[#0000]: File examples/tuple.er, line 4
4│ q = (1, 1.0)
         ^
SyntaxError: invalid syntax
Error[#0000]: File examples/tuple.er, line 7
7│ i, j = 0, 1
    ^
SyntaxError: invalid syntax
Error[#0000]: File examples/tuple.er, line 7
7│ i, j = 0, 1
           ^
SyntaxError: invalid syntax
```

**Comment by mtshiba** [comment]
Tuple is not yet implemented, but we have already implemented arrays. Type checking may be difficult, but modifying the parser should not be difficult.

**Comment by mtshiba** [comment]
Fixed in #90 and #91.
